src/keepers/delayedOffchainOrders.ts

```diff
@@ -154,7 +154,6 @@ export class DelayedOffchainOrdersKeeper extends Keeper {
       });
       const tx = await this.market.executeOffchainDelayedOrder(account, priceUpdateData, {
         value: updateFee,
-        gasLimit: 15_000_000,
       });
```

Comment:
Just a note here. When you call a method directly like this ethers will estimate gas for you. Many of Synthetix contracts doesn't estimate gas correctly, so on the frontend we something like this:
```
const gasLimit = await market.estimateGas.executeOffchainDelayedOrder(account, priceUpdateData, { value:updateFee })
const tx = market.executeOffchainDelayedOrder(account, priceUpdateData, {
     value: updateFee,
     gasLimit: gasLimit.mul(1.2),
})
```
Maybe this method is fine, but worth keeping in mind if you see "Out of gas" errors